db/log_test.cc

```diff
@@ -436,6 +436,17 @@ TEST_P(LogTest, BadRecordType) {
   ASSERT_EQ("OK", MatchError("unknown record type"));
 }
 
+TEST_P(LogTest, IgnorableRecordType) {
+  Write("foo");
+  // Type is stored in header[6]
+  SetByte(6, static_cast<char>(kRecordTypeSafeIgnoreMask + 1));
```

Comment:
If we use these in order, I recommend something like `+ 100` instead of `+ 1`

Reply:
Fixed